extensions/wikia/AdEngine/js/lookup/prebid/adaptersRegistry.js

```diff
@@ -81,8 +84,19 @@ define('ext.wikia.adEngine.lookup.prebid.adaptersRegistry', [
 		});
 	}
 
+	function getPriority() {
+		var priority = [];
```

Comment:
We're using it as a dictionary. Please define it as an object to avoid type conversion at first assignment.

Reply:
Done, thx.